match/lib/match/utils.rb

```diff
@@ -58,6 +58,12 @@ def self.get_cert_info(cer_certificate_path)
       return {}
     end
 
+    def self.is_cert_valid?(cer_certificate_path)
+      cert = OpenSSL::X509::Certificate.new(File.binread(cer_certificate_path))
+      now = Time.now.utc
+      (now <=> cert.not_after) == -1
```

Comment:
Same here, also would like to see an explicit `return` assuming we're using the return value

Reply:
I added an explicit return for readability
i have simplified the test to test only end date, is it better like this? (more readable, respectful of the coding style ?)